MSCatalog/Public/Save-MSCatalogUpdate.ps1

```diff
@@ -22,6 +22,9 @@ function Save-MSCatalogUpdate {
         .PARAMETER UseBits
         If using a Windows system you can use this parameter to download the update using BITS.
 
+        .PARAMETER AcceptMultiFileUpdates
+        Allow for multiple file update downloads to occur without user interaction
```

Comment:
Can we change this to "If specified, updates containing multiple files will not prompt the user to select which file to download. Instead all files will be downloaded."